src/transformers/visitors/toOrchestrationVisitor.ts

```diff
@@ -953,6 +953,7 @@ const visitor = {
             ],
             interactsWithSecret: true,
           });
+          if (indicator.isStruct) newNode.declarations[0].isStruct = true;
```

Comment:
sorry quick note - is this a repeated line of the one below? or do we need it?

Reply:
maybe it got added while resolving the merge conflicts